lib/ruby_terraform/commands/workspace.rb

```diff
@@ -3,21 +3,20 @@
 module RubyTerraform
   module Commands
     class Workspace < Base
```

Comment:
I've never been too happy with the way this command works since it's actually a collection of commands. I think it might be cleaner to break them out and mixin anything that is common. This is an aside rather than a request as it's probably out of the scope of this PR.